src/components/backend-ai-tasker.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ @license
+ Copyright (c) 2015-2020 Lablup Inc. All rights reserved.
+ */
+
+import {css, customElement, html, LitElement, property} from "lit-element";
+
+class Task {
+  taskid: string;
+  taskobj: Object;
+  status: string;
+  created_at: number;
+
```

Comment:
What about adding `finished_at` property as well, for possible future use before garbage collection.

Reply:
Fixed at 6aa5765